.github/CONTRIBUTION.md

```diff
@@ -1,18 +1,25 @@
+# Guiding
+
 ## Developing
 
 1. Install with `yarn`
 1. Transpile code: `yarn build` (or `yarn build:watch`)
 1. Link to Sense Desktop (locally on your computer):
```

Comment:
We shouldn't have this option here really. since you can't do this unless you have access to sense-client

Reply:
Good point. Let me remove those lines